### DISCUSSION TITLE
No traffic after rekeying

### DISCUSSION THREAD

**Discussion by sergiud** [strongswan/strongswan#686] [Q&A]
Before I explain my problem, let me to note that the symptoms are similar to those described in #670.

I have the following split tunnel setup in my `/etc/ipsec.conf`:
```ini
conn my
	eap_identity=%identity
	esp=aes256-sha1
	ike=aes256-sha256-prfsha256-modp1024
	leftauth=eap-mschapv2
	leftdns=%config4
	leftid=<id>
	leftsourceip=%config4
	leftupdown=/etc/ipsec.user
	#rekey=no
	right=<server>
	rightauth=pubkey
	rightca=<csca>.cer
	rightfirewall=yes
	rightid="CN=<server>"

conn my-split
	also=my
	auto=route
	dpdaction=hold
	rightsubnet=<edited>/16
```
with the updown script `/etc/ipsec.user`:
```bash
#!/bin/bash

shopt -s extglob

case "$PLUTO_VERB" in
    @(up|down)-client)
	if [[ $PLUTO_VERB =~ ^up ]]; then
	    action=-A # append
        else
	    action=-D # delete
	fi
        iptables -t nat ${action} POSTROUTING -s "$PLUTO_MY_SOURCEIP4_1" -o <eth> -m policy --dir out --pol ipsec -j ACCEPT
        iptables -t nat ${action} POSTROUTING -s "$PLUTO_MY_SOURCEIP4_1" -o <eth> -j MASQUERADE
        ;;
esac
```
After rekeying, there's no traffic and in the log I can see only informational requests. However, I cannot ping the server or the like. One workaround that seems to work is disabling rekeying altogether. Is there something I can do to avoid this?

**Comment by tobiasbrunner** [comment]
Find out what the actual problem is first. Please provide the stuff listed in #196 if you don't want to do that yourself. If it's the same thing as in #670, though, there is nothing you can do but disabling rekeying if the other end can't be fixed.
